<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SampleView</class>
 <widget class="QMainWindow" name="SampleView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>&amp;Name:</string>
      </property>
      <property name="buddy">
       <cstring>nameLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="activeLabel">
      <property name="text">
       <string>&amp;Active:</string>
      </property>
      <property name="buddy">
       <cstring>activeCheckBox</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="activeCheckBox"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QListView" name="eventsListView"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="eventsLabel">
      <property name="text">
       <string>&amp;Events:</string>
      </property>
      <property name="buddy">
       <cstring>eventsListView</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>&amp;Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resultButton">
        <property name="text">
         <string>Get &amp;Result</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_Actions">
    <property name="title">
     <string>&amp;Actions</string>
    </property>
    <addaction name="actionShow_Tabs"/>
    <addaction name="actionShow_Settings"/>
    <addaction name="separator"/>
    <addaction name="actionUseless_Input"/>
    <addaction name="actionAdd_Files"/>
    <addaction name="actionAdd_Color"/>
    <addaction name="actionShow_Progress"/>
   </widget>
   <addaction name="menu_Actions"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionUseless_Input">
   <property name="text">
    <string>Another &amp;Input</string>
   </property>
  </action>
  <action name="actionAdd_Files">
   <property name="text">
    <string>Add &amp;Files</string>
   </property>
  </action>
  <action name="actionShow_Tabs">
   <property name="text">
    <string>Show &amp;Tabs</string>
   </property>
  </action>
  <action name="actionShow_Settings">
   <property name="text">
    <string>Show &amp;Settings</string>
   </property>
  </action>
  <action name="actionAdd_Color">
   <property name="text">
    <string>Add &amp;Color</string>
   </property>
  </action>
  <action name="actionShow_Progress">
   <property name="text">
    <string>Show &amp;Progress</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
